### ISSUE TITLE
Error in update to v1.20.1

### ISSUE THREAD

**Issue by Diogo-Rossi** [oleg-shilo/codemap.vscode#82]
When trying to install update, VS Code shows the following message:
```
Error while updating 'CodeMap' extension. Please check the [log](command:workbench.action.showWindowLog) for more details.
```
The log show the details:
```
2023-03-18 14:49:50.565 [error] PackageIntegrityCheckFailed: Signature: PackageIntegrityCheckFailed
    at yt.download (vscode-file://vscode-app/d:/Softwares/VSCode/resources/app/out/vs/code/node/sharedProcess/sharedProcessMain.js:91:468)
    at async X.x (vscode-file://vscode-app/d:/Softwares/VSCode/resources/app/out/vs/code/node/sharedProcess/sharedProcessMain.js:91:20760)
    at async X.h (vscode-file://vscode-app/d:/Softwares/VSCode/resources/app/out/vs/code/node/sharedProcess/sharedProcessMain.js:91:18897)
```

Tested in 3 different PCs.
All with portable version o VS Code.

**Comment by oleg-shilo** [comment]
Working on it

-----------

Interestingly enough, downloading the extension from marketplace and installing it manually works just fine:

![image](https://user-images.githubusercontent.com/16729806/226146327-c80832ec-c250-40f9-ac1c-49cc66dc48e9.png)

But the VSCode extension manager fails. On portable and on integrated VSCode :(

**Comment by oleg-shilo** [comment]
There are some reports on similar problems: https://github.com/microsoft/vscode/issues/158193#issuecomment-1227539642

According to the thread the problem went away without them doing anything about that

**Comment by oleg-shilo** [comment]
It's quite annoying. The extension itself works just fine. You can just download and install it. Verification on Marketplace passes fine as well. The extension manager fails to download it with not very useful message "PackageIntegrityCheckFailed"
:(

**Comment by oleg-shilo** [comment]
The problem is caused by the Marketplace defect reported 3 days ago: https://github.com/microsoft/vsmarketplace/issues/619

Will update the instructions on the extension release page. 

At the moment the only way to deal with it is to install the extension manually.

**Comment by Diogo-Rossi** [comment]
The problem seems solved after the rebuild (v1.20.2). Thanks!

**Comment by oleg-shilo** [comment]
LOL, it was not solved. 
I did rebuild it with the hope that it was some uploading problem.

But v1.20.2 triggered the same problem 3h ago. And yet... I repeated the test right now and indeed v1.20.2 installs.

Anyway, I am glad it's over as it's not much I can do about it as it is a Marketplace hosting problem: both v1.20.2 and v1.20.1 are identical codebase-wise.
